### PR TITLE
Don't hardcode ~/.emacs.d

### PR DESCRIPTION
**Pull request by swsnr**
Use locate-user-emacs-file instead.

By the way, why exactly is this package better than recentf?  And you should probably use `read` instead of `load-file` to store your bookmark list…

### CODE REVIEW

**Comment by Fuco1** [comment]
Recentf can only store files, this can store any bookmark (and with bk+ that covers virtually every type of buffer imaginable). It persists bookmarks right away instead of waiting for emacs kill hook to fire (which means if you crash you lose tons of data... given my sessions last for 20 days or more it rendered recentf useless). It doesn't come with 1000 lines of silly emacs to render ugly interface.

TLDR It's better because it works the way I want :)

**Comment by wasamasa** [comment]
@lunaryorn See also [a reddit thread](http://www.reddit.com/r/emacs/comments/37gxzs/fucos_autobookmarks_like_recentf_but_better/) for the last question.
